adapters/flipp/flipp.go

```diff
@@ -0,0 +1,199 @@
+package flipp
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/buger/jsonparser"
+	"github.com/gofrs/uuid"
+	"github.com/prebid/openrtb/v17/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+var BANNER_TYPE = "banner"
```

Comment:
why we are using here upper case variables? in go for constants it is not a good practice and for var too, do we have a reason for this? I also checked code in this project and didn't find something like this

Reply:
Fixed